components/homepage/FeaturedProjects.tsx

```diff
@@ -11,9 +11,27 @@ const FeaturedProjects = () => {
           Featured <span className="underline-magic w-fit">Projects</span>
         </h2>
         <div className="flex flex-col gap-[2.25rem] md:gap-[3rem]">
-          <FeatProjectsCard />
-          <FeatProjectsCard />
-          <FeatProjectsCard />
+          <FeatProjectsCard
+            title={"Morrent - A Car Rental Application"}
+            tech={"ReactJS, Next.js"}
+            tech1={"Nodejs, MongoDB"}
```

Comment:
Why can't this go into a json file that you can loop over? That way when sanity is integrated you can pull from sanity instead of the json file and change the content of the page without editing the code.